ibc/2025/0409_galaxy.md

```diff
@@ -0,0 +1,57 @@
+# IBC Eureka vulnerabity - Galaxy - 2025-04-09
+
+## What happened?
+
+Asymmetric Research discovered a critical vulnerability in the Ethereum light client we were using on the Cosmos Hub and Lombard. We proceeded with an emergency critical upgrade on the Hub, which gave us a temporary (24h) authorization to migrate IBC Wasm light clients.
+
+## Root cause
+
+The Ethereum light client did not correctly verify the sync committee, i.e. did not verify that a pubkey is part of the committee. Additionally, an issue with pubkey verification allowed to use non-existing public keys, allowing to bypass the sync commitee signature check.
```

Comment:
I don't think we allowed a bypass of the signature check. Instead, a different sync committee's signature is checked.

```suggestion
The Ethereum light client did not correctly verify the sync committee, i.e. did not verify that each pubkey is in the committee. This allowed an attacker to trick the light client by generating a new sync committee with the same aggregate pubkey.
```